lib/src/DotenvParser.cc

```diff
@@ -0,0 +1,423 @@
+/**
+ *
+ *  @file DotenvParser.cc
+ *  @author akaahmedkamal
+ *
+ *  Copyright 2020, An Tao.  All rights reserved.
+ *  https://github.com/an-tao/drogon
+ *  Use of this source code is governed by a MIT license
+ *  that can be found in the License file.
+ *
+ *  Drogon
+ *
+ */
+
+#include "DotenvParser.h"
+
+#include <iostream>
+
+using namespace drogon;
+
+DotenvParser::DotenvParser()
+{
+}
+DotenvParser::~DotenvParser()
+{
+}
+std::string trim(const std::string &s, const char *c)
+{
+    std::string ss = s;
+    ss.erase(s.find_last_not_of(c) + 1);
+    ss.erase(0, s.find_first_not_of(c));
+    return ss;
+}
+std::vector<std::string> splitBy(const std::string &str, const char dlm)
+{
+    std::vector<std::string> parts;
+    std::stringstream stream(str);
+    std::string part;
+    while (std::getline(stream, part, dlm))
+    {
+        parts.push_back(trim(part, " \"'"));
+    }
+    return parts;
+}
+std::string parseNamespace(const std::string &line)
+{
+    std::string ns;
+    for (const auto &c : line)
+    {
+        if (c == '[')
+        {
+            continue;
+        }
+        if (c == ']')
+        {
+            break;
+        }
+        ns += c;
+    }
+    return ns;
+}
```

Comment:
For functions that's only used in this cc file. Please add `static` to prevent symbols visible from outside

Reply:
done